.travis.yml

```diff
@@ -7,17 +7,13 @@ addons:
   firefox: latest
   google-chrome: latest
 
-before_script:
```

Comment:
Please revert the `before_script` section. It's about failing and reporting issues as quick as possible to contributors.

If linting or bundlesize is failing, the `script` section won't be triggered and a contributor will get an email from Travis within 1 minute or so.

This is needed for better CX (contributor experience)

Reply:
> If linting or bundlesize is failing, the script section won't be triggered and a contributor will get an email from Travis within 1 minute or so.

With `npm test` (or `npm run build && npm run check:bundlesize && npm run check:tests && npm run check:ts`) the build will "fail fast" (i.e. upon the first failure it comes to) the same as you describe. It's debatable if this is best though; for example maybe it would be good to know the results of **all** the checks, in case there are more failures down the line to report all together (this is how test runners behave).

Reply:
Ah, the exception to this is the [second] running of the unit tests against .all.min.js. That will still run even after `npm test` fails. You can see with [this lint error I made](https://travis-ci.org/sweetalert2/sweetalert2/builds/332223614?utm_source=github_status&utm_medium=notification).

Fixing.

Reply:
Fixed. Any failed task will now prevent any further tasks from running. https://travis-ci.org/sweetalert2/sweetalert2/builds/333070755?utm_source=github_status&utm_medium=notification